### PR TITLE
ci: configure codecov

### LINKED ISSUE #387: Setup and decide for Codecov vs Coveralls
### Clear and concise description of the problem

We should decide which coverage reporting tool we should setup in our CI

### Suggested solution

Setup https://about.codecov.io/ or https://coveralls.io/

### Alternative

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Shinigami92**
closes #387

### CODE REVIEW

**Comment by Shinigami92** [comment]
I would say, let's try it out
If there is something miss-configured, we can change it later

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -95,3 +95,45 @@ jobs:
...
+        with:
+          node-version: 16
+          cache: 'pnpm'
+
+      - name: Install deps
+        run: pnpm install
+        env:
+          CYPRESS_INSTALL_BINARY: 0
+
+      - name: Install deps
```
Comment (prisis) [inline .github/workflows/ci.yml:124]: 2 times install?

**Review by prisis — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -95,3 +95,45 @@ jobs:
...
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: 'pnpm'
+
+      - name: Install deps
+        run: pnpm install
+        env:
+          CYPRESS_INSTALL_BINARY: 0
+
```
Comment (Shinigami92) [inline .github/workflows/ci.yml:123]:
```suggestion
```

**Review by prisis — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Comment by 1LISA2** [comment]
下载 (2)

**Pull request merged**